src/main/java/com/google/sps/classes/SqlConstants.java

```diff
@@ -70,6 +70,16 @@
   
   // Constant used in the forum.
   public static final int FETCH_ALL_QUESTIONS = -1;
+
+  // Constants used for inserting an answer.
+  public static final int ANSWER_INSERT_QUESTIONID = 1;
+  public static final int ANSWER_INSERT_BODY = 2;
+  public static final int ANSWER_INSERT_AUTHORID = 3;
+
+  // Constants used for inserting a follower to an answer.
+  public static final int ANSWER_FETCH_MAXID = 1;
```

Comment:
Is this used for inserting a follower to an answer? Maybe it goes in the above section?

Reply:
yes, it is used like this: the user posts an answer, so a new entry in the Answer table is created, which has a new ID being the biggest of the table, hence MAXID, and after posting, a new entry in the AnswerFollower table is created, which has to link the current user ID to the ID of the question that was just posted, the MAXID isn't used in the process of posting only the answer

Reply:
I guess my question then is why is MAXID set to 1 in this constants file?

Reply:
it is the column number from the queryResult in PostAnswerServlet, it's just we agreed to remove the _COLUMN from all of the constants, and it follows basically the same naming conventions as the rest of the SqlConstants file so I feel like it is on the same level of readability as the rest, if you read lines 90 through 94 in the servlet it becomes clear what it is, I think that none of those constants are immediately clear out of the context of their servlets

Reply:
Considering I was confused as to what this constant was for, it might not be entirely clear out of context :) If you want to keep the name as is that's fine, though the purpose of these constants should be clear out of context as well (as in, when looking at this file), so please add a clarifying comment for those that are unclear (or at the very least, this one!)

Reply:
alright, I added a comment for this and for QUESTION_FETCH_MAXID too